system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
What about to use `import platform` to test the platform (`if platform.system() == 'Linux'`) and use `findmnt` only if running on Linux, @bcoca?